pkg/machine/qemu/stubber.go

```diff
@@ -70,6 +71,15 @@ func (q *QEMUStubber) setQEMUCommandLine(mc *vmconfigs.MachineConfig) error {
 	q.Command.SetCPUs(mc.Resources.CPUs)
 	q.Command.SetIgnitionFile(*ignitionFile)
 	q.Command.SetQmpMonitor(mc.QEMUHypervisor.QMPMonitor)
+
+	if mc.CloudInit {
```

Comment:
Should we skip the ignition part if we use cloud-init?

Reply:
Making ignition optional does not have its dedicated subtask but is tracked here https://github.com/crc-org/macadam/issues/31
I made the change you suggested in this PR as this does seem to cause issues (I’ve had less success with other similar changes I tried before :)